cli/crates/server/src/bridge/resolvers.rs

```diff
@@ -31,16 +32,109 @@ struct ResolverResponse {
     rest: serde_json::Value,
 }
 
+async fn wait_until_resolver_ready(resolver_worker_port: u16, resolver_name: &str) -> bool {
+    const RESOLVER_WORKER_MINIFLARE_READY_RETRY_COUNT: usize = 30 * 4;
+    const RESOLVER_WORKER_MINIFLARE_READY_RETRY_INTERVAL: std::time::Duration = std::time::Duration::from_millis(250);
```

Comment:
The total duration should be long enough to allow the build of a resolver to finish. If a resolver depends on many external packages, it'll take time to download and build them. In fact, this could just wait infinitely…

Reply:
Actually, scratch that, I'm being silly here… Of course, this isn't actually waiting for the build, but only for miniflare to start responding to requests. My bad. Yes, this needs to be lowered.